docs/developers/v2/subgraph-info.md

```diff
@@ -0,0 +1,40 @@
+# Introduction
+
+The Graph Network is a decentralized indexing protocol for organizing blockchain data. Applications use GraphQL to query open APIs called subgraphs, to retrieve data that is indexed on the network. With The Graph, developers can build serverless applications that run entirely on public infrastructure.
+
+Yearn maintains subgraphs for each network we have yVaults on: Ethereum, Fantom, Arbitrum, and Optimism which index and organize data from Yearn smart contracts.
+
+Subgraph information is serviced by a decentralized group of server operators called Indexers. Some subgraphs are decentralized and some run on a hosted service, Yearn currently uses both. 
+
+## Ethereum
+
+- [Explorer Page](https://thegraph.com/explorer/subgraphs/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF?view=Overview)
+- Graphql Endpoint: https://gateway.thegraph.com/api/{api-key}/subgraphs/id/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF
+- [Code Repo](https://github.com/yearn/yearn-vaults-v2-subgraph)
+
+## Fantom
+- [Explorer Page](https://thegraph.com/hosted-service/subgraph/yearn/yearn-vaults-v2-optimism)
+- Graphql Endpoint: https://thegraph.com/hosted-service/subgraph/yearn/yearn-vaults-v2-optimism

```

Comment:
the title says `Fantom` but the links say `Optimism`

Reply:
@saltyfacu I took the liberty to update the fantom links.  Hopefully this is okay.  Let me know if I can help w anything.